src/pages/settings/Wallet/WalletPage/WalletPage.tsx

```diff
@@ -373,8 +391,10 @@ function WalletPage({bankAccountList, cardList, fundList, isLoadingPaymentMethod
                                             )}
 
                                             <KYCWall
-                                                onSuccessfulKYC={(_iouPaymentType, source) => navigateToWalletOrTransferBalancePage(source)}
-                                                onSelectPaymentMethod={(selectedPaymentMethod) => {
+                                                // eslint-disable-next-line @typescript-eslint/naming-convention
+                                                onSuccessfulKYC={(_iouPaymentType: string, source: string) => navigateToWalletOrTransferBalancePage(source)}
+                                                onSelectPaymentMethod={(selectedPaymentMethod: string) => {
+                                                    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
                                                     if (hasActivatedWallet || selectedPaymentMethod !== CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT) {
```

Comment:
why not `??`? (check all cases - in all of them you can use `??` instead of disabling rules)

Reply:
Is this _specific_ linter silencing needed?

The typing is: `const hasActivatedWallet: boolean`, so why would it trigger the linter?

Reply:
It seems that all 3 cases of `// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing` left in the code are related to using `||` with trivial booleans. Are they actually necessary?

Reply:
To be on the same page, I'm asking whether this specific _linter silencing_ is necessary, not whether using the `||` operator is necessary.

Reply:
In this case, aren't both sides simply of type `boolean`?